### ISSUE TITLE
Mobile Theme: .screen-reader-text should be used in favour of .assistive-text

### ISSUE THREAD

**Issue by JayGeorge** [Automattic/jetpack#1829]
# Summary of the issue

The Jetpack minieleven module is using .assistive-text rather than the [official WordPress class of .screen-reader-text](http://codex.wordpress.org/CSS)

Furthermore the .assistive-text class used has the visibility property set to hidden which is incorrect.
# Where to find

This can be found in wp-content/plugins/jetpack/modules/minileven/theme/pub/minileven/style.css:

CC @georgestephanis, as discussed

**Comment by kraftbj** [comment]
Related: https://make.wordpress.org/themes/2015/01/26/supporting-screen-reader-text/

**Comment by samhotchkiss** [comment]
We'd definitely be open to a PR from the community on this, although it doesn't line up with our current dev goals on our internal team.

**Comment by jeherve** [comment]
Closing this for now because of the lack of activity on this. We can always reopen in the future if needed.
